src/lib.rs

```diff
@@ -597,20 +609,28 @@ pub mod pallet {
 			let _ = <Pallet<T>>::do_send_votes(timestamp, 3);
 
 			// todo: calculate actual weight
-			Weight::zero()
+			<T as Config>::WeightInfo::on_initialize()
 		}
 	}
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
 		/// Registers the parachain with the Tellor controller contracts.
 		#[pallet::call_index(0)]
-		#[pallet::weight(115413000)]
+		#[pallet::weight(<T as Config>::WeightInfo::register())]
 		pub fn register(origin: OriginFor<T>) -> DispatchResult {
 			T::RegisterOrigin::ensure_origin(origin)?;
 			// Register with parachain registry contract
 			let registry_contract = T::Registry::get();
 			const GAS_LIMIT: u64 = gas_limits::REGISTER;
+			let weights = Weights {
```

Comment:
Can we add this to the RegistrationSent event? Will be easiest way to see the actual weight values sent.

Reply:
Sure, thanks